beta-src/src/webDiplomacyTheme.ts

```diff
@@ -50,10 +50,18 @@ declare module "@mui/material/styles" {
  * define constants that are to be re-used in the theme. use generic names that
  * describe their usage.
  */
+const mainColor = "#000";
+const secondaryColor = "#fff";
 const activeButtonStyle = {
-  backgroundColor: "#fff",
+  backgroundColor: secondaryColor,
   boxShadow: "0 0 2px 2px #000",
-  color: "#000",
+  color: mainColor,
```

Comment:
since we are setting color here, and it's mainColor...we can safely leave the color off box-shadow and it will inherit mainColor from color. this way, if we change mainColor, the box shadow gets the right color too